### PR TITLE
Bootstrap command will retry itself if failed

### LINKED ISSUE #87: mgit2 bootstrap should try for the second time if the repo fail
I still get this error randomly when execute `mgit bootstrap` and if it is part of the bigger task, like `yarn reset` or docs building it breaks the whole flow badly, force me to fix the structure manually.

screen shot 2018-11-02 at 12 13 21

Since we have no idea how to fix this error, I think we should make mgit try again in such cases, probably with some timeout. I believe this is because we reach the number of simultaneous connections to github.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Introduced a mechanism which allows executing `bootstrap` command once again if it ends with an error. Closes #87.

### CODE REVIEW

File: lib/commands/bootstrap.js [hunk 1]
```diff
@@ -10,17 +10,17 @@ const path = require( 'upath' );
...
 const exec = require( '../utils/exec' );
 
-module.exports = {
+const command = {
 	/**
 	 * @param {Object} data
 	 * @param {String} data.packageName Name of current package to process.
 	 * @param {Options} data.options The options object.
 	 * @param {Repository} data.repository
+	 * @param {Boolean} [data.doNotTryAgain=false] If set on true, bootstrap command won't be executed again.
```
Comment (ma2ciek) [inline lib/commands/bootstrap.js]: `set to true`?

Comment (scofalik) [reply lib/commands/bootstrap.js]: Also `true`.

**Comment by ma2ciek** [comment]
Once it happened to me also during the `mgit update` command. I'm not sure if we should patch also this command. (For now the only one used in `reset` script is `bootstrap`, so for now it should be ok)

**Comment by pomek** [comment]
`mgit update` calls `bootstrap` if the package isn't available.

**Comment by ma2ciek** [comment]
I'm not sure if we should mention in docs about this feature, but the functionality is not exposed so we can live with it.

**Pull request merged**